tools/topology/sof/pipe-highpass-capture.m4

```diff
@@ -0,0 +1,123 @@
+# Passthrough with volume Pipeline and PCM
```

Comment:
@singalsu This should probably mention highpass as well?

Reply:
Yes, it should be. I'll fix it.

Reply:
Fixed now!